development/issue_tracking.md

```diff
@@ -0,0 +1,69 @@
+# Issue Tracking
+
+OSC uses GitHub issues to track defects, discussions, and improvements. This document explains the guidelines and lifecycles.
+
+## Opening an Issue
+
+Once a defect is found or you would like to suggest an improvement or start a discussion, the first step is to look at existing issues to see if it is already being tracked. If the issue is not being tracked, create a new one.
```

Comment:
Can we point out to existing issues mark down doc or page ?. or like knowledge base.